frontend/src/components/message.tsx

```diff
@@ -59,12 +61,12 @@ export const MessageComponent = ({
         <img src={icon} className={styles.iconStyle} />
         <p className={styles.messageStyle}>{content}</p>
       </div>
-      {role == Role.Bot && false && (
+      {report && (
```

Comment:
We could potentially make this more generic, i.e. `openSidePanel` with the button text variable being named `openSidePanelButtonText`, this would prepare us for when the datagrid piece comes through - unless @mic-smith  wanted to make this refactor in his own work?

Reply:
Yes, I'm happy to make this change in my PR as I need to edit this anyway